Power.java

```diff
@@ -0,0 +1,37 @@
+public class Power {
+
+  public static void main(String[] args) {
+
+    double power0 = 0;
+    double power1 = 0;
```

Comment:
These variables names should be changed to what they actually represent. One is the base and one is the index.